pkg/oc/lib/graph/kubegraph/edges.go

```diff
@@ -134,12 +133,23 @@ func AddAllManagedByControllerPodEdges(g osgraph.MutableUniqueGraph) {
 		case *kubegraph.ReplicationControllerNode:
 			AddManagedByControllerPodEdges(g, cast, cast.ReplicationController.Namespace, cast.ReplicationController.Spec.Selector)
 		case *kubegraph.ReplicaSetNode:
-			AddManagedByControllerPodEdges(g, cast, cast.ReplicaSet.Namespace, cast.ReplicaSet.Spec.Selector.MatchLabels)
+			selector := make(map[string]string)
+			if cast.ReplicaSet.Spec.Selector != nil {
+				selector = cast.ReplicaSet.Spec.Selector.MatchLabels
+			}
+			AddManagedByControllerPodEdges(g, cast, cast.ReplicaSet.Namespace, selector)
```

Comment:
:/ we will eventually do this in a more generic way (rather than an ever-increasing switch) right?

Reply:
Hehe, we'll need to figure something out, for sure.